### PR TITLE
Made changes to separate mother zeir_id from child zeir_id

### PR DESCRIPTION
**Pull request by bennsimon**
Added identifier search filters for opensrp_id, simprints_guid
Fixed Tests

Co-authored-by: Martin Ndegwa <ndegwamartin@gmail.com>

### CODE REVIEW

File: src/test/java/org/opensrp/web/utils/SearchHelperTest.java [hunk 1]
```diff
@@ -91,7 +79,7 @@ public void testGetContactPhoneNumberParamReturnsCorrectValueForParam() {
 	@Test
 	public void testGetChildIndentifierFromMotherClient() {
 		
-		String childSystemId = SearchHelper.getChildIndentifierFromMotherClient(testClient, "M_ZEIR_ID", "mother");
+		String childSystemId = SearchHelper.getChildIdentifierFromMotherClient(testClient, "M_ZEIR_ID", "mother");
```
Comment (ndegwamartin) [inline src/test/java/org/opensrp/web/utils/SearchHelperTest.java]: Maybe we don't need this method any more, we can remove both the test and it's implementation

File: src/main/java/org/opensrp/web/utils/SearchHelper.java [hunk 2]
```diff
@@ -303,8 +301,8 @@ public static String getRelationalId(Client c, String relationshipKey) {
 		return null;
 	}
 	
-	public static String getChildIndentifierFromMotherClient(Client client, String motherIdentifier,
-	                                                         String relationshipKey) {
+	public static String getChildIdentifierFromMotherClient(Client client, String motherIdentifier,
```
Comment (ndegwamartin) [inline src/main/java/org/opensrp/web/utils/SearchHelper.java]: We might not need this method now

File: src/test/java/org/opensrp/web/utils/SearchHelperTest.java (hunk 1 above)
Comment (bennsimon) [reply src/test/java/org/opensrp/web/utils/SearchHelperTest.java]: yeah, removed it

File: src/main/java/org/opensrp/web/utils/SearchHelper.java (hunk 2 above)
Comment (bennsimon) [reply src/main/java/org/opensrp/web/utils/SearchHelper.java]: yeah

**Pull request merged**